js/main.js

```diff
@@ -11,14 +11,19 @@ Site = {
     });
 
     $(document).ready(function () {
-
+      if ($('body').hasClass('page-info')) {
+        _this.Info.init();
+      }
     });
 
   },
 
   onResize: function() {
     var _this = this;
 
+    if (_this.Info.$team.length) {
```

Comment:
that would also allow you to check against a cached boolean rather than an object each time. not sure if much quicker but would look cleaner